### PR TITLE
Update Rails to 6.0.3.2

### PR DESCRIPTION
**Pull request by pudinha**

### CODE REVIEW

**Review by thehenster — COMMENTED** [review]
💯 for adding all the comments too. It's just less stuff to look at on the next upgrade..

File: config/application.rb [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+# require "active_model/railtie"
+# require "active_job/railtie"
 require "active_record/railtie"
+# require "active_storage/engine"
 require "action_controller/railtie"
 require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
 require "action_view/railtie"
 require "action_cable/engine"
```
Comment (thehenster) [inline config/application.rb:14]: I can't see where actioncable is being used in this app. Could potentially be removed I guess.. if that's in scope of the upgrade..

File: config/cable.yml [hunk 2]
```diff
@@ -2,9 +2,9 @@ development:
   adapter: async
 
 test:
-  adapter: async
+  adapter: test
```
[the remark this answers is not in the document]
Comment (pudinha) [reply config/cable.yml:5]: maybe it's new on Rails 6? It was added by `rails app:update`

File: bin/setup [hunk 3]
```diff
@@ -1,30 +1,36 @@
...
 
-  puts "\n== Preparing database =="
-  system! 'bin/rails db:setup'
+  # puts "\n== Copying sample files =="
+  # unless File.exist?('config/database.yml')
+  #   FileUtils.cp 'config/database.yml.sample', 'config/database.yml'
+  # end
 
-  puts "\n== Preparing elasticsearch =="
-  system! 'bin/rails search:reindex'
```
[the remark this answers is not in the document]
Comment (pudinha) [reply bin/setup:26]: whoops :eyes:

File: config/application.rb (hunk 1 above)
Comment (pudinha) [reply config/application.rb:14]: hmm true. I'm gonna take it out and see what happens

File: config/cable.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (thehenster) [reply config/cable.yml:5]: I had a rummage - it seems the test adapter is an extension of the async adapter.. https://github.com/rails/rails/blob/fbe2433be6e052a1acac63c7faf287c52ed3c5ba/actioncable/lib/action_cable/subscription_adapter/test.rb

**Review by thehenster — APPROVED** [review]
The build is failing :( But looking good..

**Comment by pudinha** [comment]
> The build is failing :( But looking good..

Sry, it was rubocop. It just needed an underscore and it should now be green.

File: config/environments/production.rb [hunk 4]
```diff
@@ -14,32 +14,28 @@
...
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
 
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV["RAILS_SERVE_STATIC_FILES"].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
```
Comment (EGiataganas) [inline config/environments/production.rb:27]: Can you please revert this?

**Review by Rhian — APPROVED** [review]
Approved on the basis that the suggested changes have now been made.

**Pull request merged**